Diary/Diary/DiaryService.cs

```diff
@@ -0,0 +1,220 @@
+            }
+        }
+
+        internal void ChangeParametrs()
+        {
+            int numberTaskChange = SelectNumberTask();
+            Console.WriteLine("enter new parameters");
+            var taskParametrs = AddTask();
+            CheckParametrs(taskParametrs, numberTaskChange);
+        }
+
+        private static string[] AddTask()
+        {
+            {
+                string text = Console.ReadLine();
+                string[] words = text.Split(new char[] { ',' });
+                return words;
+            }
+        }
+        private static void CheckParametrs(string[] taskParametrs, int numberTask)
+        {
+            switch (taskParametrs.Length)
+            {
+                case 4:
+                    Record4Parametrs(taskParametrs, numberTask);
+                    break;
+
+                case 3:
+                    Records3Parametrs(taskParametrs, numberTask);
+                    break;
+
+                case 2:
+                    Records2Parametrs(taskParametrs, numberTask);
+                    break;
+
+                case 1:
+                    Records1Parametr(taskParametrs, numberTask);
+                    break;
+
+                default:
+                    Console.WriteLine("incorrect parameter format");
+                    break;
+            }
+        }
+
+        private static void Records1Parametr(string[] taskParametrs, int numberTask)
+        {
+            PriorityTask diaryTask = new(taskParametrs[0]);
+            weeklyTasks[numberTask] = diaryTask;
+        }
+
+        private static void Records2Parametrs(string[] taskParametrs, int numberTask)
+        {
+            PriorityTask diaryTask = new(taskParametrs[0],
+                                                   (taskParametrs[1]));
+            weeklyTasks[numberTask] = diaryTask;
+
+        }
+
+        private static void Records3Parametrs(string[] taskParametrs, int numberTask)
+        {
+            PriorityTask diaryTask = new(taskParametrs[0],
+                                                      DateTime.Parse(taskParametrs[1]),
+                                                      DateTime.Parse(taskParametrs[2]));
+            weeklyTasks[numberTask] = diaryTask;
+            diaryTask.GetAlarm();
+        }
+
+        private static void Record4Parametrs(string[] taskParametrs, int numberTask)
+        {
+            PriorityTask diaryTask = new(taskParametrs[0],
+                                                                          DateTime.Parse(taskParametrs[1]),
+                                                                          DateTime.Parse(taskParametrs[2]),
+                                                                          taskParametrs[3]);
+            weeklyTasks[numberTask] = diaryTask;
+            diaryTask.GetAlarm();
+        }
+
+
+        private static string CheckTime(int i)
+        {
+            if (weeklyTasks[i - 1].timeTask == default(DateTime))
```

Comment:
`i` means index, why do you subtract 1? That means it is not index it is a task number. Name the parameter accordingly.